oe_multilingual.module

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal hooks.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Template\Attribute;
+use Drupal\Core\TypedData\TranslatableInterface;
+
+/**
+ * Implements hook_language_fallback_candidates_alter().
+ *
+ * - Disable default language fallback logic: the default Drupal behavior
+ * will return all available languages and then choose the first one
+ * ordered alphabetically.
+ * - Return the entity's original language as the default fallback language.
+ */
+function oe_multilingual_language_fallback_candidates_alter(array &$candidates, array $context): void {
+  $operation = $context['operation'];
+
+  // The default fallback language for any entity is the language it was
+  // originally written on.
+  if ($operation == 'entity_upcast' || $operation == 'entity_view') {
```

Comment:
Are we sure these are the only operations possible?
Also, use the identical operator.

Reply:
This are the two operations that are involved in the rendering of an entity. For the defined use case we only need to alter the language fallback when rendering nodes.